org.ghidra_sre.Ghidra.json

```diff
@@ -50,9 +50,9 @@
             "build-commands": [
                 "for i in docs Extensions Ghidra GPL licenses server support ; do install -d $i /app/lib/ghidra/ ; cp -r $i/ /app/lib/ghidra/$i/ ; done",
                 "install -Dm755 ghidraRun /app/lib/ghidra/ghidraRun",
-                "icotool -x support/ghidra.ico --index=8",
-                "install -Dm644 ghidra_8_256x256x32.png /app/share/icons/org.ghidra_sre.Ghidra.png",
+                "icotool -x support/ghidra.ico -i 8",
                 "install -Dm644 org.ghidra_sre.Ghidra.desktop /app/share/applications/org.ghidra_sre.Ghidra.desktop",
+                "install -Dm644 ghidra_8_256x256x32.png /app/share/icons/org.ghidra_sre.Ghidra.png",
```

Comment:
This also seems to be the exact same line that got removed. Maybe this whole commit can be dropped?

Reply:
It can. Order doesn't matter here.